### PR TITLE
Code-Review

### PR DESCRIPTION
**Pull request by Have-Samuel**
Practicing Code review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
...hqTWuE5koz4Vr3xGkqY8dBxC9VNPdXBoV6F2aoQLb5phqILPTJephzlPS1AqfyhUGvOsCs/8icjafndqX6iJLPkAaJCr7YKQbkDtxQv/1yKk6MJxfUPBvnAetYvhPlkm6a8IWWVM8tHVnIZE5Q5yZ6hd/Z1wnJyXleMknEUdiB0djywq1PZJHM2Y1FOpjpKItXZk9R0S3KQm2KTZl0a15M86SX+iXlYxIeCTtiDbf68gqQk7SdJDHjAxg5XuOFHMfEmuEd9JbBsSJd/FQ/XuongH6Rilo4y8JDewnH10Ujb6lXL8pxfIhQ4DSSEkRsIPfjcISo1NBclr19VHU6QzXBvhDnW38Vg3UqDppjrLjE0b89oQJ4+Ha7tgNImt+z7pBdrSNTTEM7UjVPHSMJYlXYuakNQ69M+bWuJ8s74JUL82TcWldbTLnHxM5sfmRyTLZ5tRr9cbbWZ5c8Od0WzunWaOpBFy3nKCVVqvu2ntLyzpTWy3T0yfA+F9iTOYtHptNlbM6TufXCxpGdpLCsi8A3nuYGB4C5f9sS1FLguxh6yR3Bi+HUVOsNsw84X6jqiijWeRnV/1kgqbHRrdH1H33Gldv+gWYNOU9u8L6tk7d1JvH2mwk2hv98SEBVf1PaZQTvnXaXvrh4Z/AznvvKG4QkeKn7fRbsuRM3sDOxwVckgvYrbmr6ZzaeLYU5S5IM7A7MyufCZ+R+V9Vy95kmmvfx8n8hSAd8HaydCtXgU712xJE4Bg/VThvPfUEFifRmXN74MQbJoNLU/ijpWTRa9++iaYna4aX23Pd6vFreZkMfstNA5XPEfR70y0ZtLm/W1RjIpi2290+//woDYSWJrtX6XDe2h7Qdh+NStx8i1EORvvYVVHkFvNpezIzop1aDX0wkxjT04dgmkkKEuzDzB4dqHewE9ydoxuwEuH0y+WjG3267rwGsdWMwtuLk2Hfjwb5x5sHPtBthAi+cjsa/b5KfznTyVd6H/Pvh57fNl0D8uFTutxilXvi3xVyPX1WAyCGkBYk51XdLz0PDiH3bQbQjsCr76L1DK0G9CPa2fVRmAYPzCMHxjGDwzjB4bxA8P4gWH8wDB+YBg/MIwfGMYPDOMHhvHz8w3VLGhJcvxhhoexSlyz9gAAAAAAAAAAAPDNf/LrRrIXyTumAAAAAElFTkSuQmCC" alt="voice-sound">
```
Comment (Have-Samuel) [inline index.html:17]: Please try to use shorter image sources, because it makes the code easier to read. :+1:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
```
Comment (Have-Samuel) [inline index.html:1]: I suggest that next time as you make your PR, please check for the Linter errors to make the code clean of errors.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    <iframe class="watch" width="560" height="315" src="https://www.youtube.com/embed/W6NZfCO5SIk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    <ul>
+        <li>JavaScript Course for Beginners</li>
+    </ul>
+    <iframe class="watch" width="560" height="315" src="https://www.youtube.com/embed/PlxWf493en4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    <ul>
+        <li>JavaScript Tutorial for Beginners</li>
+    </ul>
+    <iframe class="watch" width="560" height="315" src="https://www.youtube.com/embed/0afZj1G0BIE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    <ul>
```
Comment (Have-Samuel) [inline index.html:61]: I proppose you to try and use just a single UL element so you could cover the whole section and you just add a few properties in your css.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        <li>HTML Tutorial</li>
+    </ul>
+    </aside>
+    <footer id="footer">
+
+    </footer>
```
Comment (Have-Samuel) [inline index.html:67]: Please try to avoid empty elements in your code so it's not conjested and that makes it easy to read.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,83 @@
+body {
+  margin: 0;
+}
+#header {
+  width: 100%;
+  margin: 10px 10px 10px 10px;
+  background: lightgreen;
```
Comment (Have-Samuel) [inline style.css:7]: Please for the property Background-color should be complete, if you only want the background color bacause this avoids some errors on your page.

**Review by Have-Samuel — COMMENTED** [review]
Changes Required :
Nice work done, Keep working almost there :+1: 

FInd my comments below and don't hesitate to leave any comments.

**Review by keroloussamy — APPROVED** [review]
Hi @Have-Samuel, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- Good commits.✔️
- Good explanation .✔️

clap


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**